TrafficCapture/trafficReplayer/src/main/java/org/opensearch/migrations/replay/TrafficReplayer.java

```diff
@@ -49,7 +50,7 @@ public static JsonTransformer buildDefaultJsonTransformer(String newHostName,
             joltJsonTransformerBuilder = joltJsonTransformerBuilder.addAuthorizationOperation(authorizationHeader);
         }
         var joltJsonTransformer = joltJsonTransformerBuilder.build();
-        return new CompositeJsonTransformer(joltJsonTransformer, new TypeMappingJsonTransformer());
+        return new CompositeJsonTransformer(joltJsonTransformer, new TypeMappingJsonTransformer(), new SigV4ExcisionJsonTransformer());
```

Comment:
This should NOT strike the existing auth header, especially when it was just added via the addAuthorizationOperation above (line 50).